app/Http/Controllers/BuildNoteController.php

```diff
@@ -1,49 +1,37 @@
 <?php
-/*=========================================================================
-  Program:   CDash - Cross-Platform Dashboard System
-  Module:    $Id$
-  Language:  PHP
-  Date:      $Date$
-  Version:   $Revision$
 
-  Copyright (c) Kitware, Inc. All rights reserved.
-  See LICENSE or http://www.cdash.org/licensing/ for details.
-
-  This software is distributed WITHOUT ANY WARRANTY; without even
-  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
-  PURPOSE. See the above copyright notices for more information.
-=========================================================================*/
-
-namespace CDash\Controller\Api;
+namespace App\Http\Controllers;
 
 use App\Models\BuildNote;
+use App\Services\PageTimer;
 use App\Services\TestingDay;
-
-use CDash\Database;
 use CDash\Model\Build;
+use Illuminate\Http\JsonResponse;
 
-class ViewNotes extends BuildApi
+/**
+ * NOTE: The "build note" functionality is distinct from the "user note" functionality.
+ */
+class BuildNoteController extends AbstractBuildController
```

Comment:
`final class`?

Reply:
Yup, I did it again....  I will look into ways to automate this check.